doc/src/manual/running-external-programs.md

```diff
@@ -60,6 +60,56 @@ julia> open(`less`, "w", STDOUT) do io
 3
 ```
 
+## Cmd Object
```

Comment:
This section now seems redundant with the combination of the introduction of `Cmd` above and the first paragraph in the Execution Environment section just below?